src/bundle/ui-dev/src/modules/common/taggify/taggify.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+
+import Icon from '../icon/icon';
+import Tag from '../tag/tag';
+
+const ENTER_CODE = 'Enter';
+const COMMA_CODE = 'Comma';
+
+const Taggify = ({ tagsValue, hotKeys, allowDuplicates, onTagsChange, bottomHint }) => {
+    const [tags, setTags] = useState([]);
+    const [newTagContent, setNewTagContent] = useState('');
+    const newTagContentInputRef = useRef(null);
+    const addTag = (content) => {
+        if ((!allowDuplicates && isDuplicated(content)) || !content) {
+            return;
+        }
+
+        setTags((prevTags) => {
+            const lastTag = prevTags[prevTags.length - 1];
+            const nextId = lastTag ? lastTag.id + 1 : 0;
```

Comment:
Maybe we can start using `at`? 🤔 
ref. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at
```suggestion
            const nextId = prevTags.at(-1)?.id ?? 0;
```